<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TNMCurve</class>
 <widget class="QWidget" name="TNMCurve">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>118</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TNM Curve</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="curveControl_grp">
     <property name="title">
      <string>Editable</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="1" column="0" colspan="2">
       <widget class="QCheckBox" name="interpolateFwd">
        <property name="text">
         <string>Interpolate moves forward (CTRL+F)</string>
        </property>
        <property name="shortcut">
         <string>Ctrl+F</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QCheckBox" name="interpolateBwd">
        <property name="text">
         <string>Interpolate moves backward (CTRL+B)</string>
        </property>
        <property name="shortcut">
         <string>Ctrl+B</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="toIdentity_btn">
        <property name="text">
         <string>To Identity</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>curveControl_grp</tabstop>
  <tabstop>interpolateFwd</tabstop>
  <tabstop>interpolateBwd</tabstop>
  <tabstop>toIdentity_btn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
